lib/Onyx.js

```diff
@@ -1060,6 +1070,15 @@ function removeNullValues(key, value) {
  * @returns {Promise}
  */
 function set(key, value) {
+    if (!ActiveClientManager.isClientTheLeader()) {
+        Broadcast.sendMessage({type: METHOD.SET, key, value});
```

Comment:
It would be ok in terms of separation of concerns, but still all the tabs would have to be connected to both channels (sending the data and leader election; we cannot subscribe only the leader to the data channel and unsubscribe the others).